bin/spiced/src/main.rs

```diff
@@ -85,7 +85,7 @@ fn init_tracing() -> Result<(), Box<dyn std::error::Error>> {
         EnvFilter::new(env_log)
     } else {
         EnvFilter::new(
-            "spiced=INFO,runtime=INFO,flightsql_datafusion=INFO,flight_datafusion=INFO,sql_provider_datafusion=INFO",
+            "spiced=INFO,runtime=INFO,secrets=INFO,flightsql_datafusion=INFO,flight_datafusion=INFO,sql_provider_datafusion=INFO",
```

Comment:
Included `secrets` as this module was not previously configured so tracing didn't work